Application of linting rules

**Problem description**
Running linting rule against current API specification gives the following results:

```
 152:18  warning  camara-properties-descriptions  Property description is missing or empty: "SendCodeBody.description" property must be truthy      components.schemas.SendCodeBody
 162:22  warning  camara-properties-descriptions  Property description is missing or empty: "SendCodeResponse.description" property must be truthy  components.schemas.SendCodeResponse
 169:22  warning  camara-properties-descriptions  Property description is missing or empty: "ValidateCodeBody.description" property must be truthy  components.schemas.ValidateCodeBody
 200:15  warning  camara-properties-descriptions  Property description is missing or empty: "ErrorInfo.description" property must be truthy         components.schemas.ErrorInfo
 217:16  warning  camara-properties-descriptions  Property description is missing or empty: "two_legged.description" property must be truthy        components.securitySchemes.two_legged

✖ 5 problems (0 errors, 5 warnings, 0 infos, 0 hints)
```

Results of yamllint:
one-time-password-sms.yaml
```
  8:193     error    trailing spaces  (trailing-spaces)
  11:24     error    trailing spaces  (trailing-spaces)
  95:11     error    wrong indentation: expected 12 but found 10  (indentation)
  149:11    error    wrong indentation: expected 12 but found 10  (indentation)
  158:47    error    trailing spaces  (trailing-spaces)
  223:98    error    trailing spaces  (trailing-spaces)
```

**Expected behavior**
Errors needs to be corrected and warnings reviewed if corrections are needed.
 After implementing corrections, the Github Action can be configured in the repository to check each PR.

**Alternative solution**
<!-- A clear and concise description of any alternative solutions if any. -->

**Additional context**
<!-- Add any other context of the considered correction. -->

--- a/code/API_definitions/one-time-password-sms.yaml
+++ b/code/API_definitions/one-time-password-sms.yaml
@@ -225,7 +225,7 @@ components:
         clientCredentials:
           tokenUrl: https://auth.example.com/token
           scopes:
-            one-time-password-sms:send-validate: Permission to send OTP by SMS and to validate it  
+            one-time-password-sms:send-validate: Permission to send OTP by SMS and to validate it
   responses:
     Generic400:
       description: Problem with the client request

--- a/code/API_definitions/one-time-password-sms.yaml
+++ b/code/API_definitions/one-time-password-sms.yaml
@@ -5,10 +5,10 @@ info:
     Service Enabling Network Function API to send short-lived OTPs (one time passwords) to a phone number via SMS and validate it afterwards, in order to verify the phone number as a proof of possession.
 
     # Relevant  Definitions and concepts
-    - **NaaS**: *Network-as-a-Service* model where Telco Network resources are exposed to third parties through APIs. In this particular API, One Time Password is exposed following this model. 
+    - **NaaS**: *Network-as-a-Service* model where Telco Network resources are exposed to third parties through APIs. In this particular API, One Time Password is exposed following this model.
     - **OTP**: *One Time password* is a one-time authorization code (OTAC) that is valid for only one login session or transaction.
 
-    # API Functionality 
+    # API Functionality
     It enables a Service Provider (SP) to send an OTP code by SMS and validate it to verify the phone number (MSISDN) as a proof of possession.
 
     # Resources and Operations overview
@@ -92,7 +92,7 @@ paths:
           $ref: '#/components/responses/Generic504'
       security:
         - two_legged:
-          - one-time-password-sms:send-validate
+            - one-time-password-sms:send-validate
   /validate-code:
     post:
       tags:
@@ -146,20 +146,22 @@ paths:
           $ref: '#/components/responses/Generic504'
       security:
         - two_legged:
-          - one-time-password-sms:send-validate
+            - one-time-password-sms:send-validate
 components:
   schemas:
     SendCodeBody:
+      description: Structure to request sending a code to a phone number
       type: object
       properties:
         phoneNumber:
           $ref: '#/components/schemas/PhoneNumber'
         message:
-          $ref: '#/components/schemas/Message'         
+          $ref: '#/components/schemas/Message' 
       required:
         - phoneNumber
         - message
     SendCodeResponse:
+      description: Structure to provide authentication identifier
       type: object
       properties:
         authenticationId:
@@ -167,6 +169,7 @@ components:
       required:
         - authenticationId
     ValidateCodeBody:
+      description: Strcuture to request code verification
       type: object
       properties:
         authenticationId:
@@ -198,6 +201,7 @@ components:
       maxLength: 10
       example: AJY3
     ErrorInfo:
+      description: Structure to describe error
       type: object
       required:
         - status
@@ -216,6 +220,7 @@ components:
   securitySchemes:
     two_legged:
       type: oauth2
+      description: oauth2 security schema
       flows:
         clientCredentials:
           tokenUrl: https://auth.example.com/token

